test/html/test.ml

```diff
@@ -198,7 +205,12 @@ let () =
     (* Running the actual commands for the test. *)
     let run_test_case () =
       generate_html ();
-      validate_html html_file;
+
+      match running_in_travis, found_tidy with
+        | true, false -> Alcotest.fail "Could not find `tidy` in PATH!";
+        | _, true -> validate_html html_file;
+        | _, _ -> ();
```

Comment:
FYI, this currently breaks all the tests locally.

In particular when the matched pattern is `false, true` (second line), only the HTML validation will be run. This has to do with the OCaml's precedence rules. The content after this line is included in the last case after `(); ...`.

Reply:
Maybe we should consider running ocamlformat on odoc :)